packages/react-hooks/src/useThresholds.ts

```diff
@@ -18,6 +16,49 @@ export type ThresholdApi = {
   slashProposalThreshold: number;
 }
 
+const PROPOSE_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 1,
+  [KUSAMA_GENESIS]: 0.5,
+  [POLKADOT_GENESIS]: 0.6,
+  default: 0.5
+};
+
+const REJECT_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 0.5,
+  [KUSAMA_GENESIS]: 0.5,
+  [POLKADOT_GENESIS]: 0.5,
+  default: 0.5
+};
+
+const SLASH_THRESHOLDS: Record<string, number> = {
+  [KUSAMA_GENESIS]: 0.5,
+  [POLKADOT_GENESIS]: 0.75,
+  default: 0.5
+};
+
+const TREASURY_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 0.5,
+  [KUSAMA_GENESIS]: 0.6,
+  [POLKADOT_GENESIS]: 0.6,
+  default: 0.6
+};
```

Comment:
Sorry if I made you do this -

I was only suggesting moving out the functions, not the config itself. The reason for keeping the config in-place is that teams actually do adjust these. (There are actually 2 more that should have made their adjustments by now, but haven't as of yet)

TL:DR Functions here, but do export the const from the apps-config package :)

Reply:
Ok, hope it's ok now :)